apps/web/src/views/Swap/V3Swap/containers/SwapCommitButton.tsx

```diff
@@ -197,19 +208,15 @@ export const SwapCommitButton = memo(function SwapCommitButton({
   // End Modals
 
   const onSwapHandler = useCallback(() => {
-    if (isExpertMode) {
-      handleSwap()
-    } else {
-      setSwapState({
-        tradeToConfirm: trade,
-        attemptingTxn: false,
-        swapErrorMessage: undefined,
-        txHash: undefined,
-      })
-      onPresentConfirmModal()
-    }
+    setSwapState({
```

Comment:
what should be the expert mode like in the new flow?

Reply:
yes